client/state/data-layer/wpcom/experiments/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Internal Dependencies
+ */
+import makeJsonSchemaParser from 'lib/make-json-schema-parser';
+import schema from './schema.json';
+import { EXPERIMENT_FETCH } from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { registerHandlers } from 'state/data-layer/handler-registry';
+import { assignToExperiments } from 'state/experiments/actions';
+import config from 'config';
+
+/**
+ * Transform the result from the API into the action we can use
+ *
+ * @param data The result from the API
+ * @returns {{variations: object, nextRefresh: number}} The transformed result
+ */
+const transformApiRequest = data => ( {
+	variations: data.variations,
+	nextRefresh: data.ttl,
```

Comment:
Shouldn't this be a function of the current time? Something like `now + ttl`?

Reply:
Yep, good point!